### DISCUSSION TITLE
bucket quota been set on the old version of Minio,  will it take effect after the upgrade

### DISCUSSION THREAD

**Discussion by chenshi5012** [minio/minio#17567] [Q&A] [answered]
https://github.com/minio/minio/pull/14206

Hi， @harshavardhana  

please check this pr,   if bucket quota (FIFO mode) been set on the old version of Minio,  will it take effect after the upgrade?

use ILM expiration   Can only mange the stroage file by time periodically ，  can not mange the stroage by quota.  is there any alternative solutions？

**Comment by chenshi5012** [comment]
Hi， @poornas @vadmeste  @klauspost   May I ask for your advice,

**Reply by klauspost** [reply]
Please do not ping people - if you want personal, instant support, look into our [subscriptions offers](https://min.io/pricing).

**Comment by harshavardhana** [comment, marked as answer]
> #14206
> 
> Hi， @harshavardhana
> 
> please check this pr, if bucket quota (FIFO mode) been set on the old version of Minio, will it take effect after the upgrade?
> 
> use ILM expiration Can only mange the stroage file by time periodically ， can not mange the stroage by quota. is there any alternative solutions？

No it will not take effect.

**Reply by chenshi5012** [reply]
thanks
